src/en/wip-storage.md

```diff
@@ -31,19 +31,21 @@ juju storage volume list
 
 As previously mentioned, a charm which requires storage will automatically allocate the default storage (unit filesystem) by default. It is possible to instead specify the storage to be used when the service is deployed, using constraints.
 
-The constraints can specify the type/pool and size of the storage required.
+The constraints can specify the type/pool and size, and optionally a count, of the storage required.
 
 ```
-juju deploy <charm> --storage <label>=<pool>,<size>
+juju deploy <charm> --storage <label>=<pool>,<size>[,count]
 ```
 
-For example, to deploy the postgresql service and have it use the unit’s local filesystem for 10 Gigabytes of storage for its ‘data’ storage requirement:
+Unless specified, the count is 1; this means that only one instance of the specified store will be created.
```

Comment:
It's definitely 1. If you don't specify any storage constraints for a store with min=0, then it won't create anything. But if you do specify constraints without count, it'll default to just creating 1.